beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -92,3 +79,31 @@ func (s *Server) GetAggregateAttestation(w http.ResponseWriter, r *http.Request)
 		}}
 	network.WriteJson(w, response)
 }
+
+// SubmitContributionAndProofs publishes multiple signed sync committee contribution and proofs.
+func (s *Server) SubmitContributionAndProofs(w http.ResponseWriter, r *http.Request) {
+	var req SubmitContributionAndProofsRequest
+	if r.Body != http.NoBody {
```

Comment:
isn't this nobody check required for all post requests? how do we do this for all. maybe it's not even needed it just needs to go through the decode